web/packages/teleport/src/Integrations/status/AwsOidc/Cards/ConsoleCard.tsx

```diff
@@ -0,0 +1,175 @@
+              <Icons.CircleCheck
+                size="small"
+                color="interactive.solid.success.default"
+              />
+              <Text
+                color="interactive.solid.success.default"
+                typography="body3"
+              >
+                Enabled
+              </Text>
+            </Chip>
+          </Flex>
+          <P2 mb={2} color="text.slightlyMuted">
+            Sync AWS IAM Roles Anywhere Profiles with Teleport
+          </P2>
+        </Box>
+        <Flex gap={3} my={3}>
+          <Box>
+            <Text fontWeight="500" fontSize={9} mb={2} css={{ lineHeight: 1 }}>
+              {profiles}
+            </Text>
+            <H3>Profiles</H3>
+            <P3 color="text.muted" mb={1}>
+              AWS Roles Anywhere Profiles are available on the Resources Page.
+            </P3>
+            <Box borderTop={1} borderColor="interactive.tonal.neutral.0">
+              <Text
+                color="text.muted"
+                fontSize={2}
+                fontWeight="300"
+                mb={2}
+                mt={1}
+                css={{ fontStyle: 'italic' }}
+              >
+                All {groups} Groups are being synced
+              </Text>
+            </Box>
+          </Box>
+          <Box>
+            <Text fontWeight="500" fontSize={9} mb={2} css={{ lineHeight: 1 }}>
+              {roles}
+            </Text>
+            <H3>Roles</H3>
+            <P3 color="text.muted" mb={1}>
+              AWS IAM Roles are available as dropdown options for the respective
+              Profiles.
+            </P3>
+            <Box borderTop={1} borderColor="interactive.tonal.neutral.0">
+              <Text
+                color="text.muted"
+                fontSize={2}
+                fontWeight="300"
+                mb={2}
+                mt={1}
+                css={{ fontStyle: 'italic' }}
+              >
+                Filtered By: {filters.join(', ')}
+              </Text>
+            </Box>
+          </Box>
+        </Flex>
+        <SyncStamp date={updated} />
+      </Flex>
+    </CardTile>
+  );
+}
+
+function EnrollCard() {
+  return (
+    <CardTile width="100%" data-testid={`console-enroll`}>
+      <Flex flexDirection="column" justifyContent="space-between" height="100%">
+        <Box>
+          <Flex alignItems="center">
+            <H2>AWS Console and CLI Access</H2>
+          </Flex>
+          <P2 mb={2} color="text.slightlyMuted">
+            Create new app resources to access your AWS account.
+          </P2>
+        </Box>
+        <Flex alignItems="center" gap={2}>
+          <H3>Enable Access</H3>
+          <Icons.ArrowForward />
+        </Flex>
+      </Flex>
+    </CardTile>
+  );
+}
+
+const Chip = styled(Flex).attrs({
+  flexDirection: 'row',
+  alignItems: 'center',
+  gap: 1,
+})`
+  padding: ${({ theme }) =>
+    `${theme.space[1]}px ${theme.space[3]}px ${theme.space[1]}px ${theme.space[2]}px`};
```

Comment:
It might be easier to specify those as (IIRC) `pl`, `pr`, `pt`, `pb` attrs a couple of lines above.

https://github.com/gravitational/teleport/blob/b8327e0b8e2a727f0ff6030141f21136b7d3d35e/web/packages/teleport/src/DeviceTrust/EmptyList.tsx#L97